src/dataset-builder/CohortEditor.test.ts

```diff
@@ -436,31 +444,23 @@ describe('CohortEditor', () => {
     expect(screen.getByText('Group 1')).toBeTruthy();
   });
 
-  it.skip('can add/delete criteria groups and maintain user understandable names', async () => {
+  it('can delete criteria groups and maintain user understandable names', async () => {
     // Arrange
     showCohortEditor();
     const user = userEvent.setup();
-    // Assert
-    expect(screen.getByText('Group 1')).toBeTruthy();
     // Act
     await user.click(screen.getByText('Add group'));
-    // Assert
-    expect(screen.getByText('Group 2')).toBeTruthy();
-    // Act
-    await user.click(screen.getByLabelText('delete group 2'));
     await user.click(screen.getByText('Add group'));
```

Comment:
If you change the test to no longer delete group 2, then this is adding group 3, which isn't needed until after the assert on line 456.

Reply:
the deletion of group 2 happens below on line 259, then the test confirms that group 3 is now called group 2